### PR TITLE
phpcs-and-linting-issues branch

### PR DESCRIPTION
**Pull request by gudmdharalds-a8c**

### CODE REVIEW

**Comment by gudmdharalds-a8c** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 554e49099e0dd333520b7da445911fe9956ef4f6 ([view code](https://github.com/gudmdharalds-a8c/vip-go-ci-manual-testing/tree/554e49099e0dd333520b7da445911fe9956ef4f6)).

***

This bot provides automated PHP linting and [PHPCS scanning](https://docs.wpvip.com/technical-references/code-review/phpcs-report/). For more information about the bot and available customizations, see [our documentation](https://docs.wpvip.com/technical-references/code-review/vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected `;`


https://github.com/gudmdharalds-a8c/vip-go-ci-manual-testing/blob/554e49099e0dd333520b7da445911fe9956ef4f6/file-10.php#L3

**Comment by gudmdharalds-a8c** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 554e49099e0dd333520b7da445911fe9956ef4f6).

**Comment by gudmdharalds** [comment]
**PHP Syntax Errors Found**


Scan performed on the code at commit 554e49099e0dd333520b7da445911fe9956ef4f6 ([view code](https://github.com/gudmdharalds-a8c/vip-go-ci-manual-testing/tree/554e49099e0dd333520b7da445911fe9956ef4f6)).

***

This bot provides automated PHP linting and [PHPCS scanning](https://docs.wpvip.com/technical-references/code-review/phpcs-report/). For more information about the bot and available customizations, see [our documentation](https://docs.wpvip.com/technical-references/code-review/vip-code-analysis-bot/).

***

**Error**: syntax error, unexpected `;`


https://github.com/gudmdharalds-a8c/vip-go-ci-manual-testing/blob/554e49099e0dd333520b7da445911fe9956ef4f6/file-10.php#L3

File: file-11.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+print_r( 'test' );
```
Comment (gudmdharalds) [inline file-11.php:3]: :warning: **Warning( severity 5 )**: print_r() found. Debug code should not normally be used in production (*WordPress.PHP.DevelopmentFunctions.error_log_print_r*).

File: file-11.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+echo time() . "testing" . PHP_EOL;
```
Comment (gudmdharalds) [inline file-11.php:5]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: file-11.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+echo esc_html( $n );
```
Comment (gudmdharalds) [inline file-11.php:7]: :warning: **Warning( severity 3 )**: Variable $n is undefined (*VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable*).

**Review by gudmdharalds — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

:warning: 2 warnings 

***


This bot provides automated PHP linting and [PHPCS scanning](https://docs.wpvip.com/technical-references/code-review/phpcs-report/). For more information about the bot and available customizations, see [our documentation](https://docs.wpvip.com/technical-references/code-review/vip-code-analysis-bot/).

**Pull request closed without merging**